mne/beamformer/tests/test_rap_music.py

```diff
@@ -45,7 +45,7 @@ def _get_data(ch_decim=1):
     return evoked, noise_cov
 
 
-def simu_data(evoked, forward, noise_cov, n_dipoles, times, nave=1):
```

Comment:
any reason why you delete `n_dipoles` here? I can see it is set to 2 in the test below, but might still be nice to have the flexibility moving forward? Or does it actually only support max. 2 sources?

Reply:
simu_data, in its current state, produces exactly 2 sources so I've decided to remove 'n_dipoles'.